.eslintrc.js

```diff
@@ -3,14 +3,14 @@ module.exports = {
   "env": {
     "mocha": true,
     "node": true,
+    "browser": true,
   },
   "plugins": ["chai-friendly"],
   "rules": {
-    "no-unused-expressions": 0,
     "chai-friendly/no-unused-expressions": 2,
-    "browser": true,
     "import/no-extraneous-dependencies": 0,
     "import/extensions": 0,
     "import/no-unresolved": 0,
+    "jsx-quotes": [2, "prefer-single"],
```

Comment:
Basically it requires single quotes for JSX attributes. So for example, if I'm setting `class` I would use `class='someclass'` rather than `class="someclass"`.